streams/src/test/java/org/apache/kafka/streams/integration/IQv2VersionedStoreIntegrationTest.java

```diff
@@ -168,16 +167,13 @@ private void shouldHandleVersionedKeyQuery(final Optional<Instant> queryTimestam
                                                final Long expectedTimestamp,
                                                final Optional<Long> expectedValidToTime) {
 
-        VersionedKeyQuery<Integer, Integer> query = VersionedKeyQuery.withKey(RECORD_KEY);
-        if (queryTimestamp.isPresent()) {
-            query = query.asOf(queryTimestamp.get());
-        }
-
-        final StateQueryRequest<VersionedRecord<Integer>> request = StateQueryRequest.inStore(STORE_NAME).withQuery(query).withPositionBound(PositionBound.at(INPUT_POSITION));
-        final StateQueryResult<VersionedRecord<Integer>> result = IntegrationTestUtils.iqv2WaitForResult(kafkaStreams, request);
+        // define query
```

Comment:
Comment can be removed. We call `defineQuery()` -- very obvious what is happening (what also means, it's well structured and easy to understand code)